magefiles/rulesengine/repos/release_service_catalog.go

```diff
@@ -59,7 +59,7 @@ var ReleaseServiceCatalogRepoSetDefaultSettingsRule = rulesengine.Rule{Name: "Ge
 		//This is env variable is specified for release service catalog
 		os.Setenv(fmt.Sprintf("%s_CATALOG_URL", rctx.ComponentEnvVarPrefix), fmt.Sprintf("https://github.com/%s/%s", rctx.PrRemoteName, rctx.RepoName))
 		os.Setenv(fmt.Sprintf("%s_CATALOG_REVISION", rctx.ComponentEnvVarPrefix), rctx.PrCommitSha)
-
+		os.Setenv("DEPLOY_ONLY", "application-api dev-sso enterprise-contract has pipeline-service ingresscontroller integration internal-services release")
```

Comment:
So this will apply to all the release test suites that are run on ephemeral clusters? We don't build on any of them, all of them reuse existing artifacts?

Reply:
Yes, I think it only will matter for the ephemeral clusters as we don't provision stage for each test

Reply:
> Some components which must be installed don't need to be set to the variable.

What does this mean? How does it work then? Are you saying that this is not the final set that will be used? Something else will be deployed as well? How is that determined?

Also, I don't think you answered my question - can you confirm that we don't need build service for any of the tests that run in ephemeral clusters?

Reply:
> > Some components which must be installed don't need to be set to the variable.
> What does this mean? How does it work then? Are you saying that this is not the final set that will be used? Something else will be deployed as well? How is that determined?

I mean there is some logic in the installation script which ensures some components being installed without choice.  

> Also, I don't think you answered my question - can you confirm that we don't need build service for any of the tests that run in ephemeral clusters?

We don't need build service in ephemeral clusters currently.  If we want to build image in each test run again, we need to change it.

Reply:
OK, thanks.